db/seeds/activity.rb

```diff
@@ -85,3 +85,6 @@
     activity.save!
   end
 end
+
+report = Report.where.not(state: :approved).find_or_create_by!(organisation_id: delivery_partner.id, fund_id: gcrf_fund.id)
+report.active!
```

Comment:
I'm not sure what the effect of calling `find_or_create_by!` on a `where.not` relation is. It might be more reliable to use the abstractions already in the application and do something like this:

```rb
Report.for_activity(first_project).create!(state: :active)
```

Reply:
Happy to go with your suggestion - reset my DB with it in place and all looks good, I suppose I would like to see this in its own `reports` seed file, but I thinnk we can live without.

I'll make that change and then merge this.